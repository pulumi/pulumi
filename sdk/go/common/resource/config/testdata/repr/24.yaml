my:outer:
  inner:
    secure: securevalue
