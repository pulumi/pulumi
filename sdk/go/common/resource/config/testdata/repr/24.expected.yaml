decrypt:
  my:outer: '{"inner":{"secure":"securevalue"}}'
paths:
  my:outer:
    value:
      inner:
        secure: securevalue
    string: '{"inner":{"secure":"securevalue"}}'
    redacted: '{"inner":"[secret]"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABF/4JCAAEFaW5uZXIXbWFwW3N0cmluZ11pbnRlcmZhY2Uge33/gh8AAQZzZWN1cmUGc3RyaW5nDA0AC3NlY3VyZXZhbHVl
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:outer.inner:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
